item_content/item.rb

```diff
@@ -0,0 +1,28 @@
+require 'date'
+class Item
+  attr_accessor :genre, :author, :source, :label, :publish_date
+
+  def initialize(publish_date, source: nil, archived: true)
+    @id = Random.rand(1..1000)
+    @genre = nil
+    @author = nil
+    @label = nil
+    @source = source
+    @publish_date = Date.today || publish_date
+    @archived = archived
+  end
+
+  private
+
+  def can_be_archived?
+    @publish_date > 10
+  end
```

Comment:
- [ ] `Date.today` in ruby will return a date in this format `YYYY-MM-DD`. therefore your `can_be_archived?` won't function properly because you are comparing a date format to the number `10` instead of `10 years` as stated in the project requirement, here is a suggestion on how you can implement this. :+1: :100: 

```sh
      today = Date.today
      ten_years_ago = Date.new(today.year - 10, today.month, today.day)

      @published_date < ten_years_ago

```